docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -153,6 +153,30 @@ By default, YugabyteDB presplits an index into `ysql_num_shards_per_tserver * nu
 
 {{< /note >}}
 
+### SPLIT AT VALUES
+
+For range-sharded indexes, you can use the `SPLIT AT VALUES` clause to set split points to presplit range-sharded indexes.
+
+**Example**
+
+```plpgsql
+CREATE TABLE tbl(
+  a int,
+  b int,
+  primary key(a asc, b desc);
```

Comment:
Maybe use uppercase for `primary key`, `asc`, and `desc` here and elsewhere for consistency on this page.

Reply:
Fixed in https://github.com/yugabyte/yugabyte-db/pull/17213/commits/ed03336e8b5cb6211ff0b0fe3ad076ca2103be02